ManageEngine/adauditplus/ingest/parser.yml

```diff
@@ -84,5 +85,5 @@ stages:
           user.target.name: "{{ parse_event.message.duser }}"
           user.target.id: "{{ parse_event.message.duid.strip('%{').strip('}') }}"
 
-          observer.hostname: "{{ parse_event.message.cs3 }}"
+          event.provider: "{{ parse_event.message.cs3 }}"
```

Comment:
This change only works when `cs1=AD Analytics`. Otherwise, `cs3` represents the observer name.

Reply:
@squioc  According to the doc it's the event source 
![image](https://github.com/SEKOIA-IO/intake-formats/assets/90054310/37088277-361b-4223-b6b9-3d9872baeb33)

I don't find a pattern to create a condition between the event provider and the observer hostname... 
![image](https://github.com/SEKOIA-IO/intake-formats/assets/90054310/3c551c92-24a7-4398-a819-411737a44c83)

If you have an idea, your help is welcome :D